thoth/storages/graph/models_base.py

```diff
@@ -100,3 +100,26 @@ def get_python_package_version_index_combinations(*, index_as_property: bool = T
             )
 
     return result
+
+
+class QueryResults:
```

Comment:
I would probably name it `QueryResult` as it is a result of a query call (even though there are 2 queries under the hood) - and maybe move to another file - it does not look like it should belong to models file.